bin/informational/patron_information

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+"""Find information about a particular patron based on barcode and pin.
+"""
+
+import os
+import sys
+
+bin_dir = os.path.split(__file__)[0]
+package_dir = os.path.join(bin_dir, "..", "..")
+sys.path.append(os.path.abspath(package_dir))
+
+from core.scripts import Script
+import argparse
+from core.model import Library
+from api.authenticator import LibraryAuthenticator, PatronData
+
+class PatronInformationScript(Script):
+    @classmethod
+    def arg_parser(cls):
+        parser = argparse.ArgumentParser()
+        parser.add_argument(
+            '--library-id',
+            help="ID of the library the patron belongs to.",
+            required=True,
+        )
+        parser.add_argument(
+            '--barcode',
+            help="The barcode to lookup.",
+            required=True,
+        )
+        parser.add_argument(
+            '--pin',
+            help="The pin for barcode.",
+            default=None,
+        )
+
+        return parser
+
+    def do_run(self, _db=None, cmd_args=None, output=sys.stdout):
+        _db = _db or self._db
+        args = self.parse_command_line(_db, cmd_args=cmd_args)
+        library = Library.lookup(self._db, short_name=args.library_id)
+        if library is None:
+            print "Library does not exist."
+        else:
+            auth = LibraryAuthenticator.from_config(_db, library).basic_auth_provider
+            patron_data = auth.remote_authenticate(args.barcode, args.pin)
+            self.explain(patron_data)
+
+    def explain(self, patron_data):
+        if patron_data is None or patron_data is False:
+            print "Account does not exist."
+        else:
+            print "Permanent ID: %s" % self.print_data(patron_data.permanent_id)
```

Comment:
Generally we use `self.log.info()` instead of `print` statements. It doesn't matter much here but this makes sure the logging information goes to wherever logs are stored as well as standard output.

Now that I think about it, though, we probably _don't_ want these logs to go where logs are stored, because they contain patron information, so this is better.